GliaWidgets/Sources/CallVisualizer/ScreenSharing/View/ScreenSharingView.swift

```diff
@@ -0,0 +1,95 @@
+import UIKit
+
+extension CallVisualizer {
+    final class ScreenSharingView: BaseView {
+
+        // MARK: - Props
+
+        struct Props: Equatable {
+            let style: ScreenSharingViewStyle
+            let endScreenSharing: Cmd
+            let back: Cmd
+
+            init(
+                style: ScreenSharingViewStyle = Theme().screenSharing,
+                endScreenSharing: Cmd = .nop,
+                back: Cmd = .nop
+            ) {
+                self.style = style
+                self.endScreenSharing = endScreenSharing
+                self.back = back
+            }
+        }
+
+        // MARK: - Properties
+
+        private lazy var header = Header(with: Theme().screenSharing.header).make { header in
```

Comment:
I would suggest to pass styling via props to have single entry point for controlling view.

Reply:
Yeap, I guess I just forgot to pass it into entry point.

Reply:
[b1a8f14](https://github.com/salemove/ios-sdk-widgets/pull/470/commits/b1a8f14161e658948ceec99163d53c10ebe7778c)